docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -38,9 +38,9 @@ You can use the UID2 SDK for Python on the server side to facilitate the followi
 
 This SDK simplifies integration with UID2 for any DSPs or UID2 sharers who are using Python for their server-side coding. The following table shows the functions it supports.
 
-| Encrypt Raw UID2 to UID2 Token | Decrypt UID2 Token | Generate UID2 Token from DII | Refresh UID2 Token |
-| :--- | :--- | :--- | :--- |
-| Supported | Supported | Supported | Supported |
+| Encrypt Raw UID2 to UID2 Token | Decrypt UID2 Token | Generate UID2 Token from DII | Refresh UID2 Token | Map Personal Data to a Raw UID2 |
```

Comment:
In UID2 "Personal Data" is called DII. @genwhittTTD you can correct if I am mistaken here

Reply:
You are right, I checked UID2 java docs, it called `DII`. I updated it to `Map DII to a Raw UID2`